admin/acceptance_tests/layertree_test.py

```diff
@@ -151,15 +152,69 @@ def test_unlink(self, test_app, layertree_test_data, dbsession):
 
         assert item not in group.children
 
+    def test_new_button(self, test_app, layertree_test_data):
+        resp = self.get(test_app)
+        theme = layertree_test_data['themes'][0]
+        group = layertree_test_data['groups'][0]
+        ogc_server = layertree_test_data['ogc_servers'][0]
+        layer_wms = layertree_test_data['layers_wms'][0]
+        layer_wmts = layertree_test_data['layers_wmts'][0]
+        # no new on layers
+        for item_id in (layer_wmts.id, layer_wms.id):
+            assert 0 == len(
+                resp.html.select(
+                    'tr.treegrid-{} li.action-new a'.format('_{}_{}_{}'.
+                                                            format(theme.id, group.id, item_id))))
+        for item_id, path, new_item, title_name, required_fields in (
+            (
```

Comment:
item_id should be named parent_id ?

Reply:
+1 and new_item could be changed to route_table or something similar but more precise.

Reply:
👍